gacl/admin/object_search.php

```diff
@@ -1,6 +1,6 @@
 <?php
 require_once('gacl_admin.inc.php');
```

Comment:
regarding html escaping, wouldn't this make more send to do this in the smarty template at gacl/admin/templates/phpgacl/object_search.tpl

For example the following line at https://github.com/openemr/openemr/blob/master/gacl/admin/templates/phpgacl/object_search.tpl#L21 would be:
``````
<th colspan="2">{$object_type_name|escape:'html'} > {$section_value_name|escape:'html'}</th>
``````